packages/devtools/src/constants.ts

```diff
@@ -2,12 +2,14 @@ import {Account} from './types';
 
 export const ETHERLIME_ACCOUNTS: Account[] = [
   {
+    // This is used by Mike and Liam both on their local environments
```

Comment:
Is this still true @kerzhner?

Reply:
I use the last private key for local development. It is easy for me to switch. Do you think that it's a good idea to leave comments like this in our source code? I wonder if there is a better way to keep track of what private keys folks are using. Probably a better solution is for each developer to generate a private key + address pair locally that is never checked into source control.

Reply:
I don't think it is a good idea, no. I'll use new, unique, different keys for my local environment, CircleCI, and Heroku.

Reply:
I updated Heroku's HUB_ADDRESS to 0x37e335623C9994F2473d791d0C3adA14E422022b and updated Netlify too.
Also updated HUB_ADDRESS on config.yml for my liam-heroku-hub branch which changes what CircleCI uses.
These should be unique and won't conflict with anyone's dev environment. (edited)